### ISSUE TITLE
OpenSSL instructions - Need clarification / further explenation

### ISSUE THREAD

**Issue by jtanner-msft** [Azure/azure-iot-sdks#294]
Trying to confirm the Windows build, I found that I can get the build_all.cmd to work, but when I try to use build_all.cmd with --use-websockets it fails.  I've set the environment variable as indicated, but am not clear on how to 'install with dynamic libraries'. as is mentioned in step 1.  So far I've just performed the git checkout of that project to the indicated directory.

Install OpenSSL 1.0.1 (x86) (tip: install with dynamic libraries); 
2.Additionally to instructions on step 1 above, create the environment variable OPENSSL_ROOT_DIR=C:\OpenSSL 
3.Run the build_client.cmd script with the option  --use-websockets .

**Comment by jebrando** [comment]
The statement "install with dynamic libraries" refers to the fact that you need to build the binaries before running the build_all.cmd.  I will try to reword this to make it more clear.

**Comment by bhalgat20** [comment]
Were you able to build the Web socket version successfully on Windows ? If yes , How did yo manage to build Open SSL 1.0.1 libraries separately?

**Comment by jebrando** [comment]
https://github.com/openssl/openssl/blob/master/INSTALL

See the installation Header for your platform

**Comment by jasoneilts** [comment]
Regarding the note:
"To enable support to AMQP over WebSockets,
Install OpenSSL 1.0.1 (x86) (tip: install with dynamic libraries);
Additionally to instructions on step 1 above, create the environment variable OPENSSL_ROOT_DIR=C:\OpenSSL
Run the build_client.cmd script with the option --use-websockets."

When attempting to run build_client.cmd with the --use-websockets flag, I received an error saying it couldn't find some OpenSSL DLL's to bincopy.  After looking at the culprit (CMakeLists.txt), it attempts the file copy with the "OpenSSLDir" envvar instead of OPENSSL_ROOT_DIR.  The documentation should reflect that this environment variable also must be set in order to build correctly.

**Comment by jebrando** [comment]
The text has been changed.
